### PR TITLE
OSDOCS-6453: Supportability - install failure and sos reports

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-6453

Link to docs preview:
https://64442--docspreview.netlify.app/microshift/latest/microshift_install/microshift-installing-troubleshooting

QE review:
- N/A uses previously QE'd technical content; docs plumbing

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@pacevedom and @dhellmann could either or both of you PTAL? This mostly pulls in existing our own `sos` content, but does create a troubleshooting assembly for the Install book.

**Review by dhellmann — COMMENTED** [review]
/lgtm

**Review by xenolinux — COMMENTED** [review]
I left one suggestion but otherwise looks good!

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

File: microshift_support/microshift-sos-report.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ include::_attributes/attributes-microshift.adoc[]
 toc::[]
 
 [role="_abstract"]
-`sos` is a tool you can use to collect troubleshooting information about a host. An `sos report` will generate a detailed report with all the enabled plugins and data from the different components and applications in a system.
+`sos` is a tool you can use to collect troubleshooting information about a host. An `sos report` command generates a detailed report showing all of the enabled plugins and data from the different components and applications in a system.
```
Comment (abhatt-rh) [inline microshift_support/microshift-sos-report.adoc:10]: Per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#general-text):

> Do not start a sentence with a lowercase term; rewrite the sentence.

For example:
```suggestion
You can use the`sos` tool to collect troubleshooting information about a host. An `sos report` command generates a detailed report showing all of the enabled plugins and data from the different components and applications in a system.
```

**Review by abhatt-rh — COMMENTED** [review]
@ShaunaDiaz and @xenolinux: Peer review suggestions look good to me. Also, I have left a comment related to capitalization. Thanks!

**Pull request merged**